mobilecoind/clients/python/jupyter/wallet.ipynb

```diff
@@ -82,7 +82,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "monitor_id = client.add_monitor(credentials)"
+    "monitor_id = client.add_monitor(credentials.account_key)"
```

Comment:
It doesn't return the monitor_id, it returns an `AddMonitorResponse`, so that might be a source of confusion to someone reading this.

Reply:
Good point - fixed up and made all responses explicit